tests/integration_tests/functional/test_drive_vhost_user.py

```diff
@@ -207,8 +188,12 @@ def test_device_ordering(microvm_factory, guest_kernel, rootfs_ubuntu_22):
     fs2 = drive_tools.FilesystemFile(os.path.join(vm.fsfiles, "scratch2"), size=512)
     vm.add_drive("scratch2", fs2.path)
 
+    # Create a rw rootfs file that is unique to the microVM
+    rootfs_rw = Path(vm.chroot()) / "rootfs"
+    shutil.copy(rootfs_ubuntu_22, rootfs_rw)
```

Comment:
nit: Would it make sense to carve these two commands out to their own function or even fixture?

Reply:
To be honest, I don't expect for this to be ubiquitous as using vhost-user-blk rootfs doesn't seem like a separate dimension. We should be able to test most of the features with vhost-user-blk as an additional drive, plus it should be more efficient from the space point of view. So I don't see much value in it at this point, but we can reconsider it later.